iroh-rpc/src/lib.rs

```diff
@@ -1,12 +1,8 @@
-pub fn hello_world() -> &'static str {
-    "hello world"
-}
-
-#[cfg(test)]
-mod tests {
-    #[test]
-    fn it_works() {
-        let result = 2 + 2;
-        assert_eq!(result, 4);
-    }
-}
+mod behaviour;
+mod client;
+mod commands;
+mod error;
+mod serde;
+mod server;
+mod stream;
+mod swarm;
```

Comment:
something should probably be public, so others can use use it ;)

Reply:
it's like you think this is open source or something I want other people to use it